codec-http/src/test/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandlerTest.java

```diff
@@ -266,107 +265,108 @@ public void testCheckInvalidWebSocketPath() {
         assertNull(WebSocketServerProtocolHandler.getHandshaker(handshakerCtx.channel()));
     }
 
-    @Disabled("buffer migration")
+    @Disabled("TODO: fix me")
```

Comment:
I am figuring out why these tests are failing, that is the reason why this PR is a draft